woocommerce/woocommerce-gutenberg-products-block
# Use the <header> element for the checkout header

## What
The checkout template uses the checkout header template part, but it is set to use `<div>`

Fixes #10558

## Why

For accessibility, it should use the `<header>` element like the regular header template part.
 


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Install and activate a blocks theme eg TT3/TT4
2. Go to `Appearance` > `Editor` > `Templates` > `Manage All Templates` > `Page: Checkout` ensure none has customizations (or delete them if so), and enter edition mode
3. On `List View` select `Checkout Header`, open `Block` > `Advanced` pane and ensure `HTML ELEMENT` is set to `<header>`
4. Do a test purchase, and on the checkout page verify the header is wrapped in a `<header>` tag




## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Use the `<header>` element for the checkout header template part
